default.nix

```diff
@@ -0,0 +1,15 @@
+{ pkgs  # nixpkgs package set
+}:
+
+{
+  # script to collect all *.dump-hi files produced by the compiler
+  # into $out/dist-hi directory, preserving directory structure
+  collect-dump-hi-files = ''
+    mkdir $out/dist-hi
+    cd dist
+    find . -type f -name '*.dump-hi' -print0 | ${pkgs.cpio}/bin/cpio --pass-through --null --make-directories $out/dist-hi
```

Comment:
interesting usage of cpio, didn't know about that :)

Reply:
I learned it from Linux From Scratch, that's what they use to put specific kernel modules into initramfs image